generate-example-index.ts

```diff
@@ -0,0 +1,660 @@
+        .join("") +
+        "\n\n;" +
+        eventListeners
+    );
+
+    // Cascading Style Sheets
+    const css = formatCSS(
+      example
+        .$("style")
+        .map((_i, elem) => elem.children[0])
+        .get()
+        .map((elem): string => elem.data)
+        .join("")
+    );
+
+    // Hypertext Markup Language
+    const $html = cheerio.load(example.$("body").html());
+    $html("script").remove();
+
+    const html = formatHTML($html("body").html());
+
+    // Resources
+    const fixPath = (rawPath: string): string =>
+      /^https?:\/\//.test(rawPath)
+        ? rawPath
+        : path
+            .resolve(path.dirname(example.path) + path.sep + rawPath)
+            .replace(this._projectPath, this._webURL);
+    const resources = {
+      js: example
+        .$("script")
+        .map((_i, elem): string => cheerio(elem).attr("src"))
+        .get()
+        .map(fixPath),
+      css: example
+        .$("link[rel='stylesheet']")
+        .map((_i, elem): string => cheerio(elem).attr("href"))
+        .get()
+        .map(fixPath)
+    };
+
+    return {
+      code: {
+        css,
+        html,
+        js
+      },
+      resources
+    };
+  }
+  private _generateJSFiddle(example: Example): Cheerio {
+    const data = this._generatePlaygroundData(example);
+
+    const form = cheerio("<form>");
+    form.attr("action", "http://jsfiddle.net/api/post/library/pure/");
+    form.attr("method", "post");
+    form.attr("target", "_blank");
+    form.append(
+      cheerio("<button>")
+        .addClass("icon jsfiddle")
+        .attr("alt", "JSFiddle")
+        .attr("title", "JSFiddle")
+        .html("&nbsp;") // No break space helps align the icon better.
+    );
+
+    // JavaScript
+    form.append(
+      cheerio("<input>")
+        .attr("type", "hidden")
+        .attr("name", "js")
+        .attr("value", data.code.js)
+    );
+
+    // Cascading Style Sheets
+    form.append(
+      cheerio("<input>")
+        .attr("type", "hidden")
+        .attr("name", "css")
+        .attr("value", data.code.css)
+    );
+
+    // Hypertext Markup Language
+    form.append(
+      cheerio("<input>")
+        .attr("type", "hidden")
+        .attr("name", "html")
+        .attr("value", data.code.html)
+    );
+
+    // Resources
+    form.append(
+      cheerio("<input>")
+        .attr("type", "hidden")
+        .attr("name", "resources")
+        .attr("value", [...data.resources.css, ...data.resources.js].join(","))
+    );
+
+    // Don't run JS before the DOM is ready.
+    form.append(
+      cheerio("<input>")
+        .attr("type", "hidden")
+        .attr("name", "wrap")
+        .attr("value", "b")
+    );
+
+    // Title
+    form.append(
+      cheerio("<input>")
+        .attr("type", "hidden")
+        .attr("name", "title")
+        .attr("value", example.titles.join(" | "))
+    );
+
+    return form;
+  }
+  private _generateCodePen(example: Example): Cheerio {
```

Comment:
Maybe we also could add support for jsbin? :pray: 
https://jsbin.com/help/api/

Reply:
JS Bin seems to have API only for pro JS Bin account (which I don't have).

Reply:
Also wouldn't committing the token to a public repo be security risk? Couldn't it be abused?

Reply:
> JS Bin seems to have API only for pro JS Bin account (which I don't have).

I love jsbin and I'm using it for almost a decade I think. 
We could talk to @remy maybe he feels supportive and gives us access for free. I don't think we create any relative traffic...

Reply:
Ok, let's do this in a later step - if possible...